### PR TITLE
Kevin s/commons

### PR DESCRIPTION
**Pull request by 11TK11**

### CODE REVIEW

File: src/main/java/org/fundacionjala/pivotal/model/pageobjects/story/StoryPage.java [hunk 1]
```diff
@@ -1,13 +1,14 @@
 package org.fundacionjala.pivotal.model.pageobjects.story;
 
 import org.fundacionjala.pivotal.model.pageobjects.AbstractPage;
+import org.fundacionjala.pivotal.util.CommonActions;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.ui.ExpectedConditions;
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/pivotal/model/pageobjects/story/StoryPage.java]: MINOR Remove this unused import 'org.openqa.selenium.support.ui.ExpectedConditions'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/main/java/org/fundacionjala/pivotal/util/CommonActions.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package org.fundacionjala.pivotal.util;
+
+import org.fundacionjala.pivotal.model.pageobjects.AbstractPage;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/pivotal/util/CommonActions.java]: MINOR Remove this unused import 'org.openqa.selenium.support.ui.WebDriverWait'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/main/java/org/fundacionjala/pivotal/util/CommonActions.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    }
+    public static void waitAndClick(WebElement webElement) {
+        CommonActions commonActions = new CommonActions();
+        commonActions.waitClick(webElement);
+    }
+    private void waitSet(WebElement element, final String textToSet) {
+        wait.until(ExpectedConditions.visibilityOf(element));
+        element.sendKeys(textToSet);
+    }
+    public static void WaitAndSetText (WebElement element, final String textToSet) {
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/pivotal/util/CommonActions.java]: MINOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00100)

File: src/main/java/org/fundacionjala/pivotal/model/pageobjects/dashboard/PageFormCreate.java [hunk 4]
```diff
@@ -5,14 +5,15 @@
 import java.util.List;
 import java.util.Map;
 import org.fundacionjala.pivotal.model.pageobjects.AbstractPage;
+import org.fundacionjala.pivotal.util.CommonActions;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.ui.ExpectedConditions;
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/pivotal/model/pageobjects/dashboard/PageFormCreate.java]: MINOR Remove this unused import 'org.openqa.selenium.support.ui.ExpectedConditions'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/main/java/org/fundacionjala/pivotal/model/pageobjects/login/SignInPage.java [hunk 5]
```diff
@@ -5,13 +5,14 @@
 import org.apache.logging.log4j.Logger;
 import org.fundacionjala.pivotal.model.pageobjects.AbstractPage;
 import org.fundacionjala.pivotal.model.pageobjects.dashboard.PageDashboard;
+import org.fundacionjala.pivotal.util.CommonActions;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.ui.ExpectedConditions;
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/pivotal/model/pageobjects/login/SignInPage.java]: MINOR Remove this unused import 'org.openqa.selenium.support.ui.ExpectedConditions'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by carledriss** [comment]
SonarQube analysis reported 5 issues
* ![MINOR][MINOR] 5 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Review by ghost — APPROVED** [review]

**Pull request merged**